javascript/packages/orchestrator/src/network.ts

```diff
@@ -341,6 +356,10 @@ export class Network {
         `https://polkadot.js.org/apps/?rpc=${wsUri}#/explorer`,
       ],
       [decorators.cyan("Prometheus Link"), node.prometheusUri],
+      [
+        decorators.cyan("Log Path"),
+        this.client.tmpDir + "/" + node.name + ".log",
```

Comment:
```suggestion
        decorators.cyan( this.client.providerName === "native" ? "Log Path" : "Log Command"),
        this.client.providerName === "native" ? this.client.tmpDir + "/" + node.name + ".log" : logCommand,
```

`native` provider have local log path, for podman/k8s we should use a command to get them.

Reply:
Oh wait.. I have [fixed that but it seems something is missing from the commit](https://github.com/paritytech/zombienet/pull/649/files#diff-383c717b42d1e4d1d684c80415b0f84f1674f1e3c67ea7ebdde64dcfe1b8d882R341-R353). I dont know how it happened. I wanted to have `logCommand` there